cmd/cloudshell_open/main.go

```diff
@@ -611,12 +598,24 @@ func instrumentlessCoupon() (*instrumentless.Coupon, error) {
 
 func promptInstrumentless() error {
 	coupon, err := instrumentlessCoupon()
-	if err != nil {
-		return fmt.Errorf("could not get instrumentless coupon: %v", err)
+
+	if err != nil || coupon == nil {
+		fmt.Println(infoPrefix + " Create a new billing account:")
+		fmt.Println("  " + linkLabel.Sprint(billingCreateURL))
+		fmt.Println(questionPrefix + " " + "Once you're done, press " + parameterLabel.Sprint("Enter") + " to continue: ")
+
+		if _, err := bufio.NewReader(os.Stdin).ReadBytes('\n'); err != nil {
+			return err
+		}
+
+		return nil
 	}
 
-	fmt.Println(infoPrefix + " Apply a starter coupon to create a billing account:" +
-		"\n  " + linkLabel.Sprint(coupon.URL))
+	code := strings.TrimPrefix(coupon.URL, trygcpURL+";code=")
```

Comment:
you can probably do URL.Parse then get .Path and trim that if you want (in case base domain changes in the future etc) but this is fine for now.